### ISSUE TITLE
How to upload directory of files via mega CLI properly

### ISSUE THREAD

**Issue by dagolinuxoid** [meganz/MEGAcmd#349]
this is out of mega-cp --help
```
Usage: put  [-c] [-q] [--ignore-quota-warn] localfile [localfile2 localfile3 ...] [dstremotepath]
Uploads files/folders to a remote folder

Options:
 -c	Creates remote folder destination in case of not existing.
 -q	queue upload: execute in the background. Don't wait for it to end' 
 --ignore-quota-warn	ignore quota surpassing warning. 
                    	  The upload will be attempted anyway.

Notice that the dstremotepath can only be omitted when only one local path is provided. 
 In such case, the current remote working dir will be the destination for the upload.
 Mind that using wildcards for local paths in non-interactive mode in a supportive console (e.g. bash),
 could result in multiple paths being passed to MEGAcmd.

```
I logged  via mega-login successfully
I have *local* `photos/` folder which should be uploaded to MEGA-Sync *cloud*
By performing `mega-put photos/ photos/` I get the error: Failed to upload: invalid argument.
Uploading distinct files is working as expected `mega-put filename clouddirectory/`
I come up with the solution of using `*` so `mega-put photos/* photos` works too.
I think it's worth noting in docs (or readme manual) the usage since bare `mega-put localFolder/ remoteMegaCloudFolder` doesn't work. Or am I missing something?

**Comment by polmr** [comment]
remove the trailing `/` in the first argument.
Still, that will create a folder named photos within the remote photos (it the latter exists). You'd rather want to use `mega-put photos` without destiny or `mega-put photos ./`
We'll review the `Failed to upload: invalid argument` with the trailing `/` in the first argument though.

**Comment by dagolinuxoid** [comment]
yep, the usage you've mentioned of a single argument *without* the slash is working good. I wish my bash's auto-completion magically remove the forward slash =) . Seriously it looks like the mega cli gets confused by the forward slash (btw I'm using 14.04_amd64)

**Comment by aabellagm** [comment]
Addressed in #383
